documentation/src/docs/asciidoc/docinfos/index-docinfo.html

```diff
@@ -0,0 +1 @@
+<link rel="icon" type="image/png" href="{{ site.baseurl }}/img/junit5-logo.png">
```

Comment:
Regarding the Javadoc: I think it should be ok to add an absolute reference to `http://junit.org/junit5/assets/img/junit5-logo.png`.